File: two-sum/two-sum.py
class Solution(object):
    def twoSum(self, nums, target):
        for i in range(len(nums)-1):
            firstnum=nums[i]
            for j in range(i+1,len(nums)):
                secondnum=nums[j]
                if firstnum+secondnum == target:
                    return [i,j]
        return []
                
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        

File: two-sum/two-sum.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
    unordered_map <int,int> mp;
        for(int i=0; i<nums.size();i++){
            int num=nums[i];
            int c =target-num;
            auto it = mp.find(c);
            if (it != mp.end()){
                return{ it->second, i};
            }
            mp[num]=i;
        }
        return {};
    }
};